### PR TITLE
Update Calendar Dates

### PR DESCRIPTION
**Pull request by SylentButDedly**
- The Final Shape Release
- Salvation's Edge Raid
- Episode: Echoes Acts I, II, and III
- Solstice
- First Iron Banner of the season (I don't think they've announced any more yet)
- Marks the Zero Hour date with the Exotic Quest icon
- Adds Episode: Echoes to the seasonal view

### CODE REVIEW

**Comment by chrisfried** [comment]
Just FYI, I started backdating old Iron Banners, but man is it tedious.

File: src/app/pipes/milestone.pipe.ts [hunk 1]
```diff
@@ -51,6 +51,8 @@ export class MilestonePipe implements PipeTransform {
           case '2023-02-28':
           case '2023-5-23':
           case '2023-05-23':
+          case '2024-06-4':
```
Comment (chrisfried) [inline src/app/pipes/milestone.pipe.ts:54]: Probably needs to be 04

Comment (SylentButDedly) [reply src/app/pipes/milestone.pipe.ts:54]: Ah yea you're right, it seems to work for me but I assume it might not work for some browsers

**Comment by SylentButDedly** [comment]
@chrisfried Yea I know, I honestly wasn't sure how to find the dates without having to go through every twab they've ever written

**Comment by chrisfried** [comment]
I filtered to Iron Banner activities and just started marking the ones I'd played in. I'm sure I missed a few, but it seemed like a good starting point.

**Pull request merged**